### DISCUSSION TITLE
Content Ideas for the UTD Survival Guide

### DISCUSSION THREAD

**Discussion by WillieCubed** [acmutd/nebula-guide#1] [Ideas]
Hi there! I have a big document of things to be included in the UTD Survival Guide, but this is a request for information that people may find relevant in a handbook or guide for incoming and current college students.

Examples of good ideas:
- How University Housing works
- How to survive college on a budget
  - How to cook and eat well
  - How to stay in shape
- When to consider changing your major/minor

All thought-out ideas will be considered further, and we may reach out to you for more information.

Feel free to leave any ideas below! Be as specific as possible so we know how to best address the concern in a guide entry!

**Comment by nealgandhi** [comment]
Hi, i would love to see a comprehensive guide (either on release or in a future update) about University Housing. I see the same questions being asked frequently or questions that have answers hidden away under many layers of UTD website pages and think it would be beneficial if there was a one stop shop for all of this info. So questions like: "idk what my room number stands for", "what do i need to bring for the unfurnished apartments", "what's a good sized [insert item] to bring to fit into this apartment", "Where can i find decently priced furniture", etc. aren't brought up over and over.

**Reply by WillieCubed** [reply]
Yes, and I think we can expand on this further by incorporating this in a comprehensive guide to living on and near campus, including logistics about being a student not from the area. For example, we can include information about obtaining food or how to store personal belongings over the summer.

I definitely like this idea.

**Reply by taliadev** [reply]
I totally agree with both you and @WillieCubed on having a section about University Housing. I'm the student government residential affairs chair this year, so I will definitely try to get more information from Housing about how the housing process works and clarification on just _how_ unfurnished some units are compared to others, or full dimensions of dorm furniture. It's kind of a travesty how long it takes to search the subreddit for clarification.

Obtaining food is definitely a huge thing. I didn't realize until late fall semester that the Comet Cupboard even existed, and by then it was too late for me to get enough food to stock up for the dining-hall-less winter break. It was a long, hungry month, and if I didn't know the Comet Cruiser route and where the grocery stores were nearby, I don't know what I would have done.

**Comment by wbne** [comment]
I think it would be cool to have these two as articles:
- How to take grad classes as an undergrad (including why you might want to)
- TAMS to UTD (a more formalized set of info for those who don't have a TAMS senior friend to ask)
While there are sources for these, they are pretty hard to find since they are niche making the survival guide a great place to consolidate the information

**Reply by wbne** [reply]
Something else I realized is a page recommending what main computing device a person should get based off their major. In addition general pros/cons of using certain computing mediums like tablets or desktops.

**Comment by adityarathod** [comment]
One thing that may be difficult but important to tackle from the student perspective is the social scene/how to make friends at UTD.

People here tend to be introverted and r/utdallas posters often complain about this aspect of student life here. I think tackling the "0 to friend group" problem would have high impact.

**What would be out of scope:** teaching social skills

**What would be in scope:** general guidance on where to meet new people as a freshman

**Reply by WillieCubed** [reply]
I think this would provide a lot of value to new students, whether first-time in college freshmen, those transferring from community colleges, or other non-traditional students. One aspect of the guide I think we want to emphasize is the student perspective. Having guides for common tasks will do a great deal, but anecdotes from students will be one of this guide's killer features. I plan on getting the Mercury and other student media involved for that.

**Comment by ReshmiCode** [comment]
Students might not know which CS-guided elective to take. So a portion on the guide that lists upper-level electives and student testimonials on what they did and get out of the class might be helpful

**Comment by taliadev** [comment]
I'm not sure how well this will fit the tone that the survival guide is aiming for, but I think there should be more advice on how to handle drug and consent-related issues. Despite every freshman being assigned consent and alcohol abuse-related modules at the beginning of the year, I don't think they fully reflect the tough decisions people have to make in vulnerable situations. If your underaged friend blacks out from one-too-many shots at a party and they stop breathing, does your average student know that they can call 911 [without having to fear disciplinary action](https://studentwellness.utdallas.edu//docs/RAP.pdf)? If there's someone at the parties you frequent who constantly cajoles other people into making bad substance-related decisions, how do you call them out? How should you react if someone confides in you that they've been sexually assaulted?

These are hard questions I'd like to see answered, especially since a lot of students don't feel comfortable asking their PAs these questions.

**Reply by adityarathod** [reply]
Absolutely! I think a "questions you were probably too scared to ask" would be really valuable

**Comment by taliadev** [comment]
Something else I would like to see (maybe in the residential section?): how to cook on a budget, including in the dorms. I kept a lot of canned and dry ingredients on hand freshman year since I only had access to a minifridge, but I was able to cook a lot of nutritious and cheap meals with basically just a rice cooker and an Instant Pot.
